client/web/emberclear/app/components/app/sidebar/actions/action/index.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import Action from 'emberclear/models/action';
+
+interface IArgs {
+  action: Action;
+}
+
+export default class ResponsePanel extends Component<IArgs> {
+  get header() {
```

Comment:
this is unused -- which then makes the class empty, and then it can be deleted :D

Reply:
This is the wrong class name- it should be used in the same directory. I'm going to change that file and this one to make sure its used right.

Reply:
well, this matches the above file:
![image](https://user-images.githubusercontent.com/199018/82964642-040a9500-9f94-11ea-9892-efaeb8a6fedd.png)

which doesn't reference `this.header`

Reply:
Yeah it references the action's header, which was removed when I moved it into this class, just forgot to update that reference.